tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -27,9 +27,10 @@
 class StreamsBrokerCompatibility(Test):
     """
     These tests validates that
-    - Streams 0.11+ w/ EOS fails fast for older brokers 0.10.2 and 0.10.1
-    - Streams 0.11+ w/o EOS works for older brokers 0.10.2 and 0.10.1
-    - Streams fails fast for 0.10.0 brokers
+    - Streams works for older brokers 0.11 (or newer)
+    - Streams w/ EOS-alpha works for older brokers 0.11 (or newer)
+    - Streams fails fast for older brokers 0.10.0, 0.10.2, and 0.10.1
+    - Streams w/ EOS-beta fails fast for older brokers 2.4 or older
```

Comment:
Two test overlap and one of them is redundant -- removing the redundant one as side cleanup

Reply:
+1